### PR TITLE
PAAS-11369: Test-zuul-integration for both CI and CING

### PR DESCRIPTION
**Pull request by trungminhnguyen**
This change would be merged to have changes for "test.txt" or "test-cing.txt" at running "Test-zuul-integration", so that the updated zuul layout in ci-infra can help to run relating jobs in CI or CING.

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/30/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 6s
- zuul-tox-zuul : SKIPPED

File: integration/spec/git_repo.rb [hunk 1]
```diff
@@ -15,17 +15,17 @@ def clone_repo(host, port, user, repo)
     Git.clone("ssh://#{user}@#{host}:#{port}/#{repo}", repo)
   end
 
-  def create_test_commit
+  def create_test_commit(server)
```
Comment (hrubi) [inline integration/spec/git_repo.rb]: Variable name `server` does not make sense in this context. Please name the variable `filename`, and use it as is on line 20. Construct the filename in callers based on the `server` value.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/34/ : SUCCESS in 7s
- https://ci.intgdc.com/job/zuul-tox-zuul/40/ : FAILURE in 1m 05s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/35/ : SUCCESS in 9s
- https://ci.intgdc.com/job/zuul-tox-zuul/41/ : FAILURE in 56s

**Comment by hrubi** [comment]
Test is failing due to problems with python26 vs. python27. However, this change only updates the integration tests and does not touch zuul codebase, so I'm merging it manually.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/36/ : SUCCESS in 10s
- https://ci.intgdc.com/job/zuul-tox-zuul/42/ : FAILURE in 2m 02s

**Pull request merged**